kotlinx-coroutines-core/common/test/TestBase.common.kt

```diff
@@ -10,10 +10,10 @@ import kotlinx.coroutines.flow.*
 import kotlin.coroutines.*
 import kotlin.test.*
 
-public expect open class TestBase constructor() {
-    public val isStressTest: Boolean
-    public val stressTestMultiplier: Int
+public expect val isStressTest: Boolean
```

Comment:
Please, extract this into a separate commit and a separate PR. It would great to have it asap and separately.

Reply:
See #1579